addons/cloud_storage/static/src/core/common/composer_patch.js

```diff
@@ -0,0 +1,11 @@
+import { Composer } from "@mail/core/common/composer";
+
+import { patch } from "@web/core/utils/patch";
+import { session } from "@web/session";
+
+patch(Composer.prototype, {
```

Comment:
if this function is available for public users, then put it in the common. Otherwise you should put it under web_portal or web